app/code/core/Mage/Adminhtml/controllers/System/Convert/ProfileController.php

```diff
@@ -186,6 +186,7 @@ public function saveAction()
                     Mage::helper('adminhtml')->__('The profile has been saved.')
                 );
             } catch (Exception $e) {
+                Mage::logException($e);
```

Comment:
why add this?

Reply:
I asked myself the same question. If we solve the issue, there is no point in writing it in the exceptions file. On the other hand, if the issue is ever reproduced again, although unlikely and we are not in developer mode, it will be written there.

Reply:
I get that but the PR is to fix the trim warning so, like it's been told on other PRs, it should be mixed with other behaviors

Reply:
It is fine being removed.

Reply:
You really want a separate PR for a simple exception logging😃? 
Feel free to modify the PR, but in my opinion the exception must be logged because thats what the exception log is for.

Reply:
Either we do that for all of for nobody. And since i got told to separate when there was a specific reason for merge, we apply it here then. And anyway i dont see why adding something that’s not related to the pr

Reply:
Hm ok I don't know where you "have been told to", but when it is a rule it should be written somewhere so all contributors know it.

Secondly imho it **is** related to the PR because without it the error message about the `trim()` method would have never been visible.

Reply:
In general, pull request should be small and focused on addressing one issue or feature at a time. It just makes reviewing easier. In this case, it's probably acceptable.